api/organisations/tests/test_views.py

```diff
@@ -803,7 +847,7 @@ def test_get_subscription_metadata(
     expected_chargebee_email = "test@example.com"
 
     get_subscription_metadata = mocker.patch(
-        "organisations.models.get_subscription_metadata",
+        "organisations.models.Subscription.get_subscription_metadata",
```

Comment:
I prefer not to patch methods on concrete objects unless we really have to, is there any way to avoid it here?

Reply:
Not sure if I understood this observation. The prior version of this test was using almost the same approach but the `get_subscription_metadata` was imported from chargebee and now we are using the method with the same name in the `organisations.models.Subscription` model.

Reply:
Modify as discussed.